pkg/workflow/dataimpl.go

```diff
@@ -11,62 +14,158 @@ import (
 
 // DataImpl is the default implementation of the Data interface.
 type DataImpl struct {
-	identifier Identifier
-	header     http.Header
-	payload    interface{}
-	errors     []snyk_errors.Error
+	identifier      Identifier
+	header          http.Header
+	contentType     string
```

Comment:
Question: content-type is already stored in the header, why do we need an extra variable for it?